### PR TITLE
Don't reset the DB when using the blues flag

### PR DESCRIPTION
**Pull request by huwshimi**
Deploying models wouldn't work as the database was being cleared before the changes could be applied.

### CODE REVIEW

File: jujugui/static/gui/src/app/views/utils.js [hunk 1]
```diff
@@ -1607,25 +1607,12 @@ YUI.add('juju-view-utils', function(Y) {
       app, app.createSocketURL.bind(app, app.get('socketTemplate')),
       app.switchEnv.bind(app), app.env, model.uuid, [model], model.name,
       env => {
-        utils._detachOnLoginHandler();
```
Comment (huwshimi) [inline jujugui/static/gui/src/app/views/utils.js]: Drive-by to remove the handler that was required before we changed to the switchEnv callback.

**Comment by huwshimi** [comment]
QA:
- load `/:flags:/blues/`
- click on your username to open the profile
- create a new model by clicking the 'Create new' button
- add a service
- click 'Deploy changes'
- if you'd like you can add a credential
- scroll down and click 'Deploy'
- your model should be deployed and your app created

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/5854/
Test PASSed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/5855/
Test PASSed.

**Comment by hatched** [comment]
Thanks for this @huwshimi 

:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Comment by jujugui** [comment]
Build failed: Attempt to land pull request failed
        build url: http://ci.jujugui.org:8080/job/juju-gui-merge/1920

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge
